addons/mrp/models/mrp_production.py

```diff
@@ -106,7 +106,7 @@ def _get_default_is_locked(self):
         'Scheduled End Date',
         compute='_compute_date_planned_finished', store=True,
         help="Date at which you plan to finish the production.",
-        copy=False)
+        copy=False, readonly=False)
```

Comment:
This field is in the form and list views. I don't like setting it editable just for this fix in subcontracting. Why not bypassing the +1 hour in `_compute_date_planned_finished()` in case of subcontracting ? 
Also a quick test about that would be nice to have

Reply:
Indeed, rewrote it without changing that `readonly` property.